projects/schematics/src/add-ssr/index.ts

```diff
@@ -0,0 +1,275 @@
+import {
+  apply,
+  branchAndMerge,
+  chain,
+  externalSchematic,
+  MergeStrategy,
+  mergeWith,
+  move,
+  Rule,
+  SchematicContext,
+  SchematicsException,
+  Source,
+  template,
+  Tree,
+  url,
+} from '@angular-devkit/schematics';
+import { Schema as SpartacusOptions } from '../add-spartacus/schema';
+import {
+  addPackageJsonDependency,
+  NodeDependency,
+  NodeDependencyType,
+} from '@schematics/angular/utility/dependencies';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+import { addImport, importModule } from '../shared/utils/module-file-utils';
+import {
+  getIndexHtmlPath,
+  getPathResultsForFile,
+} from '../shared/utils/file-utils';
+import { appendHtmlElementToHead } from '@angular/cdk/schematics';
+import { experimental, strings } from '@angular-devkit/core';
+import { getProjectFromWorkspace } from '../shared/utils/workspace-utils';
+
+function addPackageJsonDependencies(): Rule {
+  return (tree: Tree, context: SchematicContext) => {
+    const dependencies: NodeDependency[] = [
+      {
+        type: NodeDependencyType.Default,
+        version: '~8.0.0',
```

Comment:
I'm wondering if we could synchronize it somehow with other (already present) angular packages.